### DISCUSSION TITLE
How to listen only on localhost?

### DISCUSSION THREAD

**Discussion by rizchaerul** [TypedProject/ts-express-decorators#1714] [Q&A] [answered]
Hi, I have a question.

If I only want to serve my backend to localhost (because 0.0.0.0 will expose my server to public if I already port forwarded port 8080 right?), how do I configure it?

![image](https://user-images.githubusercontent.com/72289631/147944277-61bf5c2b-f276-4851-8d0c-7478caa39ad9.png)

**Comment by rizchaerul** [comment, marked as answer]
Nevermind, found it on the docs https://tsed.io/docs/configuration.html#http-https-server. Sorry, should i delete this thread?

**Reply by Romakita** [reply]
No, like that those who will ask the question will find your answer :)
